code/modules/xgm/xgm_gas_mixture_presets.dm

```diff
@@ -0,0 +1,18 @@
+
+/datum/gas_mixture/vacuum/New()
+	..()
+	// does nothing, type just for clarity/organization
+
+/datum/gas_mixture/earth_standard/New()
+	..()
+	adjust_gas(GAS_OXYGEN, MOLES_O2STANDARD, 1)
+	adjust_gas(GAS_NITROGEN, MOLES_N2STANDARD, 1)
```

Comment:
- No magic numbers, that `1` is a `TRUE`
- You can probably avoid the update as it's a new mixture and you are already updating at the end anyways

Reply:
done